src/jvmTest/kotlin/PlatformComparisonTest.kt

```diff
@@ -11,8 +16,8 @@ class PlatformComparisonTest {
         val text = "a\uD83E\uDD95\uD83E\uDD96b\uDD96\uD83Ec\uD83Ed\uDD96e\uD83E"
 
         for (index in text.indices) {
-            val jvmCodePointAt = (text as java.lang.String).codePointAt(index)
-            val commonCodePointAt = CommonStringFunctions.codePointAt(text, index)
+            val jvmCodePointAt = text.codePointAt(index)
+            val commonCodePointAt = text.commonCodePointAt(index)
             assertEquals(jvmCodePointAt, commonCodePointAt, "codePointAt($index)")
```

Comment:
I was going to ask about this test. If you change its format slightly you can have the same confidence without needing the extra indirection of the actuals and the separate implementations for non-JVM targets.

Consider a test in this form:
```kotlin
@Test fun codePointAt() {
  val text = "..."
  val expected = intArrayOf(...)

  for (index in test.indices) {
    assertEquals(expected[index], text.codePointAt(index), "codePointAt($index)")
  }
}
```

This test, in `commonTest`, would retain the ability to compare the implementations by switching all platforms to compare against golden values rather than each other.

Reply:
`PlatformComparisonTest` was removed in #15